.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": ".NET 8 with .NET 6",
+    "image": "mcr.microsoft.com/dotnet/sdk:8.0",
+    "features": {
+		// https://github.com/devcontainers/features/tree/main/src/dotnet
+        "ghcr.io/devcontainers/features/dotnet:2": {
+            "version": "latest",
+            "dotnetRuntimeVersions": "6.0",
+            "aspNetCoreRuntimeVersions": "6.0",
+			"additionalVersions": "6.0"
+        }
+    },
+	"customizations": {
```

Comment:
alignment looks off in GitHub code review UI, but in code editor is aligned: 
![image](https://github.com/Azure/data-api-builder/assets/6414189/1902e194-deee-46bc-ba8d-1745dc6a1053)